docs/contributing/docs.md

```diff
@@ -0,0 +1,187 @@
+---
+title: "Crossplane Documentation"
+weight: 2000
+---
+
+- [Code of Conduct](#code-of-conduct)
+  - [Reporting](#reporting)
+- [Licensing](#licensing)
+  - [Directory structure](#directory-structure)
+  - [Run Hugo](#run-hugo)
+  - [Documentation issues and feature requests](#documentation-issues-and-feature-requests)
+- [Contributing](#contributing)
+  - [Front matter](#front-matter)
+    - [New Crossplane versions](#new-crossplane-versions)
+    - [Excluding pages from the Table of Contents](#excluding-pages-from-the-table-of-contents)
+  - [Links](#links)
+    - [Linking between docs pages](#linking-between-docs-pages)
+    - [Linking to external sites](#linking-to-external-sites)
+  - [Tabs](#tabs)
+
+## Code of Conduct
+We follow the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md).
+
+Taken directly from the CNCF CoC:
+>As contributors and maintainers in the CNCF community, and in the interest of fostering an open and welcoming community, we pledge to respect all people who contribute through reporting issues, posting feature requests, updating documentation, submitting pull requests or patches, and other activities.
+>  
+>We are committed to making participation in the CNCF community a harassment-free experience for everyone, regardless of level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, or nationality.
+
+### Reporting
+To report violations contact the Crossplane maintainers at `info@crossplane.io` or the CNCF at `conduct@cncf.io`.
+
+## Licensing
+The Crossplane documentation is under the [Creative Commons Attribution](https://creativecommons.org/licenses/by/4.0/) license. CC-BY allows reuse, remixing and republishing of Crossplane documentation with attribution to the Crossplane organization.
+
+
+### Directory structure
+The relevant directories of the docs repository are:
```

Comment:
This is the directory structure for the docs site repository right? It might be worth mentioning that we are _not_ talking about `crossplane/crossplane`.

Reply:
I updated this to be a link to `crossplane/crossplane.github.io` repo to clarify.